docs/MIGRATION_REVIEW.md

```diff
@@ -0,0 +1,231 @@
+# Sanity Migration Plan Review
+
+**Date:** November 27, 2025  
+**Reviewer:** GitHub Copilot
+
+---
+
+## Overview
+
+After reviewing the migration plan documentation in `docs/migrations/phases/`, I have compiled the following questions and suggestions organized by area.
+
+---
+
+## Questions
+
+### Phase Sequencing & Dependencies
+
+1. **Deejay/Person Schema Already Migrated?**  
+   The `person` and `dj` schemas in `/studio/schemaTypes/` suggest Phase 1's Deejay migration is partially complete. What is the current status of the other Phase 1 models (Ad, CdOfTheWeek, Concert, Music, OnDemand, Schedule, YearEndStaffPick)?
+
+2. **Cross-Phase Dependencies**  
+   Phase 5 (Year End Poll) references `YearEndStaffPick` from Phase 1. Should these be migrated together to maintain referential integrity, or is there a data relationship strategy?
+
+3. **Parallel Migration Capability**  
+   The plan says phases should be sequential, but could some models within a phase be migrated in parallel? For example, could `Ad`, `Concert`, and `Music` in Phase 1 be migrated independently?
+
+### Data Integrity & Validation
+
+4. **Soft Delete Strategy**  
+   Most MySQL tables have a `deleted` field. How will soft-deleted records be handled in Sanity? Options include:
+   - Not migrating deleted records at all
+   - Migrating them with an `isDeleted` boolean field
+   - Using Sanity's draft/published state
+
+5. **Foreign Key Relationships**  
+   Phase 4 (MRM) has foreign keys between `mrm_matches` and `mrm_bands`. How will these relationships be modeled in Sanity? Will they use Sanity references, and if so, what happens if bands are deleted?
+
+6. **Data Validation Pre-Migration**  
+   Is there a plan to validate and clean the MySQL data before migration? For example, checking for:
+   - Orphaned records
+   - Invalid URLs
+   - Duplicate entries
+   - Encoding issues in text fields
+
+### Content Migration
+
+7. **Rich Text/HTML Content**  
+   Phase 2 mentions HTML content in `CustomText` and `Story` tables. Will this be:
+   - Converted to Sanity's Portable Text format?
+   - Stored as raw HTML in a text field?
+   - Sanitized for security (XSS prevention)?
+
+8. **Image Migration Strategy**  
+   The current `importDeejays.ts` uploads images as Sanity assets. For models with `pic_url` fields pointing to external URLs (e.g., Imgur), should these be:
+   - Migrated to Sanity's asset pipeline?
+   - Left as external URL references?
+   - A hybrid approach based on domain?
+
+### Historical & Time-Sensitive Data
+
+9. **Tournament History (Phase 4)**  
+   Should historical MRM tournament data be migrated, or only the current/upcoming tournament structure? Historical data could be valuable for archives but adds complexity.
```

Comment:
We want historical data going forward. But the old data does not need to come over. The tournament runs once a year.